frontend/packages/component-library/src/breadcrumbs/stories/Breadcrumbs.story.tsx

```diff
@@ -43,6 +43,18 @@ DefaultBreadcrumbs.args = {
   ],
 };
 
+export const BreadcrumbsWithHomeIcon = SingleTemplate.bind({});
+BreadcrumbsWithHomeIcon.args = {
+  name: 'test-Breadcrumbs-home',
+  showHomeIcon: true,
+  homeLink: 'https://studio.code.org',
```

Comment:
This can be removed now that it's hardcoded. I think this is causing the `build` tests to fail.